README.md

```diff
@@ -267,7 +267,20 @@ To dump the latest 100 users with their orders:
       created_at = "desc"
 ```
 
+## Build commands
 
+For linux
+
+```zsh
+GOOS=linux GOARCH=amd64 go build -o klepto-linux
```

Comment:
`klepto-linux-amd64` to indicate target architecture. Because you might be running Linux on ARM, but this binary would not work there

Reply:
Yeah, i will add it as well

Reply:
I will add it only for linux, coz darwin runs only on amd64